src/core/ui/slider.ts

```diff
@@ -202,19 +207,24 @@ export const Slider = ($component: HTMLElement, $slider: HTMLElement, pointersLi
   };
 
   const onValueChange = (evt: MouseEvent | TouchEvent) => {
+    setEventType(evt.type);
+
+    if (evt instanceof TouchEvent && evt.touches[0]) {
+      touchClientX = evt.touches[0].clientX
+    }
 
     // find the percent [0, 100] of the current mouse position in vertical or horizontal slider
     let percent;
 
     if(type === TypeEnum.Vertical){
       const { height: boxHeight, top: boxTop } = $slider.getBoundingClientRect();
-      const mouseY = evt.type.indexOf('mouse') !== -1 ? (evt as MouseEvent).clientY : (evt as TouchEvent).touches[0].clientY;
```

Comment:
Hi,
Thank you for pr. 
It seems that this change or related one causes issues in mobile in vertical slider. 
When user drags tooltip it starts to move to the opposite direction.
Probably the issue is caused by missing "touchClientY" property.